src/main/java/io/github/moulberry/notenoughupdates/auction/APIManager.java

```diff
@@ -902,6 +904,12 @@ private CraftInfo getCraftCost(String internalname, Set<String> visited) {
 		if (recipes != null)
 			RECIPE_ITER:
 				for (NeuRecipe recipe : recipes) {
+					if (recipe instanceof KatRecipe) continue;
```

Comment:
We already have `Recipe#shouldUseForCraftCost` which is probably what you should change instead?